### PR TITLE
Start shortcut keys PR

### PR DESCRIPTION
**Pull request by just-be-dev**
This is still very much a work in progress. Hopefully will eventually solve #11.

### CODE REVIEW

**Comment by just-be-dev** [comment]
@egoist Check it out!

**Comment by just-be-dev** [comment]
So, the last items remaining are for switching between the playspots. I've started work on this. Pretty much all that's left to be done is to finish up the `playspot` model. When you press `cmd+shift+right` it'll dispatch the `playNext` action. That then needs to `commit` `UPDATE_PLAYSPOT` with the proper value for whatever the playspot should be. Same for previous.

File: src/models/layout.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+}
+
+// Computed methods
+export function leftPanelExpanded() {
+  return this.$store.state.layout.leftPanelExpanded
+}
+
+export function bottomPanelExpanded() {
+  return this.$store.state.layout.bottomPanelExpanded
```
Comment (egoist) [inline src/models/layout.js]: why not use `vuex getter`?

File: src/models/playspot.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export const UPDATE_PLAYSPOT = 'UPDATE_PLAYSPOT'
+
+export default {
+  name: 'playspot',
+  state: {
+    // TODO: Figure out how to set this initially
+    current: null
```
Comment (egoist) [inline src/models/playspot.js:8]: I guess `null` is ok, since initially there's no active route

File: src/models/playspot.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  mutations: {
+    UPDATE_PLAYSPOT(state, payload) {
+      state.current = payload
+    },
+  },
+  actions: {
+    playNext({commit}) {
+      // TODO: Get the next playspot
+      const next = null;
```
Comment (egoist) [inline src/models/playspot.js]: to get the next/previous playspot, I think we can use rootState.toys.paths, it's like:

``` js
{Button: [{path: '/Button/with_text', type: 'with text'}]}
```

You can add a getter to flatten it.

**Comment by just-be-dev** [comment]
Aaannnnd I'm done. 

Take a look!

File: src/models/playspot.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  actions: {
+    playNext({commit, getters, state}) {
+      const next = getters.playspotRoutes.indexOf(state.current) + 1
+      commit(UPDATE_PLAYSPOT, getters.playspotRoutes[next])
+    },
+    playPrevious({commit, getters, state}) {
+      const prev = getters.playspotRoutes.indexOf(state.current) - 1
+      commit(UPDATE_PLAYSPOT, getters.playspotRoutes[prev < 0 ? 0 : prev])
+    },
+    setPlayspot({commit}, payload) {
```
Comment (egoist) [inline src/models/playspot.js]: seems you didn't use this either

**Pull request merged**